runatlantis.io/docs/stats.md

```diff
@@ -9,3 +9,45 @@ Currently Statsd and Prometheus is supported. See configuration below for detail
+# HELP atlantis_cmd_autoplan_builder_execution_error atlantis_cmd_autoplan_builder_execution_error counter
+# TYPE atlantis_cmd_autoplan_builder_execution_error counter
+atlantis_cmd_autoplan_builder_execution_error 0
+# HELP atlantis_cmd_autoplan_builder_execution_success atlantis_cmd_autoplan_builder_execution_success counter
+# TYPE atlantis_cmd_autoplan_builder_execution_success counter
+atlantis_cmd_autoplan_builder_execution_success 10
+# HELP atlantis_cmd_autoplan_builder_execution_time atlantis_cmd_autoplan_builder_execution_time summary
+# TYPE atlantis_cmd_autoplan_builder_execution_time summary
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.5"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.75"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.95"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.99"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.999"} NaN
+atlantis_cmd_autoplan_builder_execution_time_sum 11.42403017
+atlantis_cmd_autoplan_builder_execution_time_count 10
+.....
+.....
+.....
+```
+Some of the important metrics which you would like to monitor are
+
+| Metric Name                                  | Metric Type                                                          | Purpose                                                                                                                                  |
+|----------------------------------------------|----------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| atlantis_cmd_autoplan_execution_error        | [counter](https://prometheus.io/docs/concepts/metric_types/#counter) | since starting of atlantis server, the number of times when [autoplan](autoplanning.html#autoplanning) on the MR’s has thrown error.     |
+| atlantis_cmd_comment_plan_execution_error    | [counter](https://prometheus.io/docs/concepts/metric_types/#counter) | since starting of atlantis server, the number of times when on commenting `atlantis plan` on the MR’s has thrown error.                  |
+| atlantis_cmd_autoplan_execution_success      | [counter](https://prometheus.io/docs/concepts/metric_types/#counter) | since starting of atlantis server, the number of times when [autoplan](autoplanning.html#autoplanning) on the MR’s has run successfully. |
+| atlantis_cmd_comment_apply_execution_error   | [counter](https://prometheus.io/docs/concepts/metric_types/#counter) | since starting of atlantis server, the number of times when on commenting `atlantis apply` on the MR’s has thrown error.                 |
+| atlantis_cmd_comment_apply_execution_success | [counter](https://prometheus.io/docs/concepts/metric_types/#counter) | since starting of atlantis server, the number of times when on commenting `atlantis apply` on the MR’s has run successfully.             |
+
```

Comment:
Please include screenshots of any notable graphs. This would encourage others to enable prom metrics :)

Reply:
again [same reason](https://github.com/runatlantis/atlantis/pull/3171#discussion_r1118095097), apologies.